symengine/cwrapper.cpp

```diff
@@ -737,14 +737,22 @@ void vecbasic_free(CVecBasic *self)
     delete self;
 }
 
-void vecbasic_push_back(CVecBasic *self, const basic value)
+CWRAPPER_OUTPUT_TYPE vecbasic_push_back(CVecBasic *self, const basic value)
 {
+    CWRAPPER_BEGIN
+
     self->m.push_back(value->m);
+
+    CWRAPPER_END
 }
 
-void vecbasic_get(CVecBasic *self, int n, basic result)
+CWRAPPER_OUTPUT_TYPE vecbasic_get(CVecBasic *self, int n, basic result)
 {
+    CWRAPPER_BEGIN
+
     result->m = self->m[n];
+
```

Comment:
Yes, exception would be thrown in both cases. @bollu Could you add the assert instead?